api/v1alpha1/flinkcluster_types.go

```diff
@@ -110,6 +110,21 @@ type JobManagerPorts struct {
 	UI *int32 `json:"ui,omitempty"`
 }
 
+// JobManagerIngress defines ingress of JobManager
+type JobManagerIngressSpec struct {
+	// Ingress host format. ex) {{$clusterName}}.example.com
+	HostFormat string `json:"hostFormat"`
+
```

Comment:
Can we make HostFormat optional? This will allow accessing the ingress with external IP address without domain name.

Reply:
I agree.